tests/conformance/README.md

```diff
@@ -0,0 +1,18 @@
+# Conformance Tests
+// TODO update readme for conformance tests after workflow is created.
+
+## Tests structure
+1. `tests/` directory contains the configuration and the test definition for conformance tests.
+2. All the conformance tests are within the `tests/conformance` directory.
+3. All the configurations are in the `tests/config` directory.
```

Comment:
Some of the configurations should be able to allow dynamic value? For example, if the test setup create a new CosmoDB collection to avoid race conditions between test runs, it should be able to return a dynamically created config file?

Reply:
This can be modified once the cosmosdb test is fixed.